client/transactions/list/test/index.js

```diff
@@ -326,4 +337,51 @@ describe( 'Transactions list', () => {
 		const { container } = render( <TransactionsList /> );
 		expect( container ).toMatchSnapshot();
 	} );
+
+	describe( 'CSV download', () => {
+		beforeEach( () => {
+			useTransactions.mockReturnValue( {
+				transactions: getMockTransactions(),
+				isLoading: false,
+			} );
+
+			useTransactionsSummary.mockReturnValue( {
+				transactionsSummary: {
+					count: 10,
+					currency: 'usd',
+					// eslint-disable-next-line camelcase
+					store_currencies: [ 'eur', 'usd' ],
+					fees: 100,
+					total: 1000,
+					net: 900,
+				},
+				isLoading: false,
+			} );
+		} );
+
+		afterEach( () => {
+			jest.resetAllMocks();
+		} );
+
+		afterAll( () => {
+			jest.restoreAllMocks();
+		} );
+
+		test( 'should render expected columns in CSV when the download button is clicked', () => {
+			render( <TransactionsList /> );
+			const downloadButton = screen.getByRole( 'button', {
+				name: 'Download',
+			} );
+
+			// simulate click on the download CSV button
+			fireEvent.click( downloadButton );
+			const expected =
+				'Transaction Id","Date / Time",Type,Amount,Fees,Net,"Order #",Source,Customer,Email,Country,"Risk level",Deposit';
+
+			// checking if columns in CSV are rendered correctly
+			expect( downloadCSVFile.mock.calls[ 0 ][ 1 ] ).toContain(
+				expected
+			);
```

Comment:
Also not critical, but IMO it's better to check exact headers set because `toContain` won't fail when new wrong header added to the export. Ideally it would be an array of expected headers, something like this: 
```js
const expectedHeaders = [ "Transaction Id", ... "Deposit" ];
const actualHeaders   = downloadCSVFile.mock.calls[0][1].split('\n')[0].split(',');
expect(actualHeaders).toEqual(expectedHeaders).
```

Reply:
Thanks, this is much better!

Reply:
Made this change in https://github.com/Automattic/woocommerce-payments/pull/1776/commits/76f3b73d2c4aa94258547e39bf057d5492a38b75

Reply:
Looks great, thank you!